crates/sui-indexer/migrations/2022-11-22-222247_error_logs/down.sql

```diff
@@ -1 +1 @@
-DROP TABLE ERROR_LOGS;
+DROP TABLE error_logs;
```

Comment:
delete this file?

Reply:
it is a separate style / typo change but that file is necessary indeed for DB setup